charts/kubescape-operator/templates/configs/cloudapi-configmap.yaml

```diff
@@ -1,6 +1,7 @@
-{{ template "account_guid" . }}
-{{ template "cluster_name" . }}
-kind: ConfigMap 
+{{- template "account_guid" . }}
```

Comment:
We should have a "configurations" section (similar to what you did with capabilities), in this configurations section we would determine if we are in "third-party mode". We would need an accountID, URL, and cluster name. 
This would prevent us from checking if we have clusterName etc.